examples/metrics/grafana-dashboards/strimzi-kafka-bridge.json

```diff
@@ -1956,7 +1956,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(strimzi_bridge_http_server_requests_total{container=~\"^.+-bridge\"}[1m])) by (method)",
+          "expr": "sum(rate(strimzi_bridge_http_server_requestCount_total{container=~\"^.+-bridge\"}[1m])) by (method)",
```

Comment:
Wondering why this metric name changed?

The original metric name is aligned to [Prometheus best practices](https://prometheus.io/docs/practices/naming/), the new one, less so.

Reply:
The metric names have changed due to vertx update (https://github.com/vert-x3/wiki/wiki/4.0.0-Deprecations-and-breaking-changes#%EF%B8%8F-full-review-of-metric-names)
However, this change is probably a typo.

Reply:
The main point really is that we screwed up in the original PR changed it in the wrong place. It should be fixed now.

Reply:
In any case good catch @LiamClarkeNZ!

Reply:
:+1: